### DISCUSSION TITLE
List of features or bugs

### DISCUSSION THREAD

**Discussion by nydevel** [denoland/deno#8835] [Q&A] [answered]
Is there are some list of features or bugs for fix/implement witch i can work on?
I just try to understand what i can start from to be helpful in Rust/TS work.

**Comment by bartlomieju** [comment, marked as answer]
You can search for issues with following tags:
- [good first issue](https://github.com/denoland/deno/labels/good%20first%20issue)
- [help wanted](https://github.com/denoland/deno/labels/help%20wanted)
- [l-rust](https://github.com/denoland/deno/labels/l-rust)
- [l-typescript](https://github.com/denoland/deno/labels/l-typescript)

**Reply by nydevel** [reply]
Thanks!
